weave-js/src/components/PagePanelComponents/Home/Browse3/filters/common.ts

```diff
@@ -40,6 +40,11 @@ export const FIELD_LABELS: Record<string, string> = {
 };
 
 export const getFieldLabel = (field: string): string => {
+  if (field.startsWith('feedback.')) {
+    // feedback.[wandb.annotation.foo].payload.value.foo --> feedback.foo
+    const bracketed = field.split('[')[1].split(']')[0];
```

Comment:
this parsing feels like something that should be centralized (alongside whatever constructs this)

Reply:
yeah good call, ill make a `parseFeedback` util and return a typed body